src/pages/Members/containers/MemberFormContainer.tsx

```diff
@@ -56,9 +57,21 @@ interface Props {
   height: number;
   id?: string;
 }
+interface CentreOptionType {
+  displayText: string;
+  id: number;
+}
 
 export default function MemberFormContainer(props: Props) {
   const { height, id } = props;
+  const isEditMode = Boolean(id);
+  const buttonTitle = isEditMode ? "Update Member" : "Create Member";
+  const titleOptions = [
```

Comment:
i think keeping this in a separate folder like "constants" would be better, just in case if any other file wants to use this. Or even keeping this in enum.

Reply:
Yes, we will need a folder and a file to store some common constants but I realized buttonTitle was not being used so I am removing it for now.